dms/security/ir.model.access.csv

```diff
@@ -3,6 +3,7 @@ id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
 access_dms_tag_user,dms_tag_user,model_dms_tag,group_dms_user,1,1,1,1
 access_dms_category_user,dms_category_user,model_dms_category,group_dms_user,1,1,1,1
 
+access_dms_storage_base_user,dms_storage_base_user,model_dms_storage,base.group_user,1,0,0,0
```

Comment:
Why adding this open permission? This seems too much.

Reply:
This change is absolutely necessary IMO.

`dms` adds read permissions for `base.group_user` to all models except `dms.storage` (`base.group_portal` for example has read permission).

The `dms_field` module and `mode="dms_tree"` widget get: storage, directories and files. A basic user without permissions in _Documents: User_ (for example an employee when accessing employee form view), not having access to `dms.storage` would have a permissions error.